webrtc.html

```diff
@@ -1516,77 +1529,101 @@
+                  </li>
+                  <li>
+                    <p>If this inspection failed for any reason, reject
+                    <var>p</var> with a <code>DOMException</code> object whose
+                    <code>name</code> attribute has the value
+                    <code>OperationError</code>, and abort these steps.</p>
+                  </li>
+                  <li>
+                    <p>Queue a task that runs the <a>final steps to create an
+                    answer</a>, given <var>p</var>.</p>
+                  </li>
+                </ol>
+                <p>The <dfn>final steps to create an answer</dfn> given a
+                promise <var>p</var> are as follows:</p>
+                <ol>
+                  <li>
+                    <p>If <var>connection</var>'s [[<a>isClosed</a>]] slot is
+                    <code>true</code>, then abort these steps.</p>
+                  </li>
+                  <li>
+                    <p>If <var>connection</var> was modified in such a way that
+                    additional inspection of the system state is necessary,
+                    then in parallel begin the <a>steps to create an answer</a>
+                    again, given <var>p</var>, and abort these steps.</p>
+                    <div class="note">
+                      This may be necessary if, for example,
+                      <code>createAnswer</code> was called when an
+                      <code><a>RTCRtpTransceiver</a></code>'s direction was
+                      "recvonly", but while performing the <a>steps to create
+                      an answer</a> in parallel, the direction was changed to
+                      "sendrecv", requiring additional inspection of video
+                      encoding resources.
+                    </div>
+                  </li>
+                  <li>
+                    <p>Given the information that was obtained from previous
+                    inspection and the current state of <var>connection</var>
+                    and its <code><a>RTCRtpTransceiver</a></code>s, generate an
+                    SDP answer, <var>sdpString</var>, as described in
+                    <span data-jsep="generating-an-answer">[[!JSEP]]</span>.
+                    </p>
+                  </li>
+                  <li>
+                    <p>Let <var>answer</var> be a newly created
+                    <code><a>RTCSessionDescriptionInit</a></code> dictionary
+                    with its <code>type</code> member initialized to the string
+                    <code>"answer"</code> and its <code>sdp</code> member
+                    initialized to <var>sdpString</var>.</p>
+                  </li>
+                  <li>
+                    <p>Unless all non-stopped
+                    <code><a>RTCRtpTransceiver</a></code>s are represented in
+                    <var>answer</var>, mark <var>connection</var> as needing
+                    negotiation.</p>
```

Comment:
I don't believe it's correct to do this here, but I'm leaving it in since it was there before. The overall issue of how negotiation-needed works is covered by issue #803.